### PR TITLE
Add configuration for OnePlus X (onyx)

### PR DESCRIPTION
**Pull request by spiiroin**

### CODE REVIEW

File: hybris.c [hunk 1]
```diff
@@ -1218,6 +1218,25 @@ led_control_breath_type(const led_control_t *self)
...
+ * - must have 'max_brightness' control file or nonzero fixed maximum
+ * - can have either
+ *    o blink on/off delay control files
+ *    o blink enabled/disable control control file
+ *
+ * Assumptions built into code:
+ *
+ * - The write() calls to sysfs controls return immediately from kernel to
+ *   userspace, but the kernel side can stay busy with the change for few
+ *   milliseconds -> Frequent intensity changes do not block mce procees,
```
Comment (izh1979) [inline hybris.c]: procees -> process

File: hybris.c [hunk 2]
```diff
@@ -1598,6 +1639,13 @@ led_control_htcvision_probe(led_control_t *self)
 
 /* ------------------------------------------------------------------------- *
  * RGB led control: binary backend
+ *
+ * One channels, which:
+ * - must have 'brightness' control file
+ *
+ * Assumptions built into code:
+ * - Using  zero brightnessdisables led, any non-zero value enables it -> If
```
Comment (izh1979) [inline hybris.c]: brightnessdisables -> brightness disables?

File: hybris.c [hunk 3]
```diff
@@ -1369,6 +1388,18 @@ led_control_vanilla_probe(led_control_t *self)
...
+ * Three channels, all of which:
+ * - must have 'brightness' control file
+ * - must have 'max_brightness' control file
+ * - must have 'on_off_ms' blink delay control file
+ * - must have 'rgb_start' enable/disable control file
+ *
+ * Assumptions built into code:
+ * - Blinking is always soft, either
+ * - The sysfs writes will block until change is finished -> Intensity changes
+ *   are slow and
```
Comment (izh1979) [inline hybris.c]: are slow and...?

**Comment by izh1979** [comment]
Few typos in comments, then LGTM. :-)

File: hybris.c (hunk 1 above)
Comment (spiiroin) [inline hybris.c]: ack

File: hybris.c (hunk 2 above)
Comment (spiiroin) [inline hybris.c]: ack

File: hybris.c (hunk 3 above)
Comment (spiiroin) [inline hybris.c]: I guess I got interrupted while writing that or something ;-)

**Comment by spiiroin** [comment]
Fixed typos.

Removed failed attempts to get "hard" blinking to work.

**Pull request merged**